corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html

```diff
@@ -0,0 +1,52 @@
+{% extends "hqwebapp/two_column.html" %}
+{% load compress %}
+{% load hq_shared_tags %}
+{% load i18n %}
+
+{% requirejs_main 'reports/v2/js/views/caseSearch' %}
+
+{% block page_sidebar_classes %}sidebar-offcanvas col-xs-12 col-sm-3 col-lg-2{% endblock %}
+
+{% block page_row_classes %}row-offcanvas row-offcanvas-left{% endblock %}
+
+{% block page_content_classes %}col-xs-12 col-sm-9 col-lg-10{% endblock %}
+
+{% block js %}{{ block.super }}
+{% endblock %}
+
+{% block stylesheets %}{{ block.super }}
+{% endblock %}
+
+{% block title %}Case Search{% endblock %}
+
+{% block page_breadcrumbs %}
+breadcrumbs todo
+{% endblock %}
+
+
+{% block page_content %}
+  {% registerurl "endpoint_filter" domain report.slug 'slug' %}
+  {% registerurl "endpoint_data" domain report.slug 'slug' %}
+
+  {% initial_page_data "report.endpoints" report.endpoints|JSON %}
+  {% initial_page_data "report.slug" report.slug %}
+<div id="case_search--model">
```

Comment:
This is a rather weird identifier - mixing underscores with dashes and having a double dash?